package.json

```diff
@@ -97,7 +97,7 @@
     "description": "Console plugin for Data Federation",
     "exposedModules": {
       "dashboard": "./packages/components/dashboard/dashboard",
-      "resourceListPage": "./packages/components/odf-resources/resource-list-page",
+      "resourceListPage": "./packages/components/resource-list-page",
```

Comment:
resource-list-page and resource-details-page should be under a directory, not directory under components...
plz add it under `components/odf-resources/` like before...

Reply:
Done